packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.css

```diff
@@ -1,75 +1,94 @@
 .content {
   z-index: var(--elevation-tooltip);
   max-width: var(--popover--width);
+  min-width: calc(var(--space-extravagant) * 3.75);
   box-shadow: var(--shadow-base);
+  margin-top: var(--space-small);
   border: var(--border-base) solid var(--color-border);
-  border-radius: var(--radius-base);
-  font-size: var(--typography--fontSize-base);
-  line-height: normal;
+  border-radius: var(--radius-larger);
+  overflow: scroll;
+  overflow-y: auto;
   background: var(--color-surface);
 }
 
 /* CAUTION: Tests for content visibility assume this class' presence hides the content
-  * so please be careful if you change this class name, or the css rules within.
- */
+* so please be careful if you change this class name, or the css rules within.
+*/
 .content.hidden {
   display: none;
   visibility: hidden;
 }
 
+.clearSearch {
```

Comment:
Can we add  ` cursor: pointer;` to this style block 👆🏻 It was historically only used when hovering over links but is now frequently used to indicate a clickable element, such as a button!

Reply:
Oh great catch! Thank you!